src/types/validationresult.rs

```diff
@@ -58,12 +76,17 @@ impl ValidationResult {
         ValidationResult::Failure(message.to_string())
     }
 
-    pub fn to_status(&self) -> UStatus {
+    pub fn to_status(&self) -> Status {
```

Comment:
You should use UStatus and not Status, I want to get rid of the googleapis dependency please

Reply:
Done - I actually missed this file during the migration.